packages/cli/openapi-ir-to-fern/src/buildTypeDeclaration.ts

```diff
@@ -519,3 +519,21 @@ function getSchemaIdOfResolvedType({
     }
     return schema;
 }
+
+function convertPropertyTypeReferenceToTypeDefinition(
+    typeReference: RawSchemas.TypeReferenceWithDocsSchema,
+    audiences: string[],
+    name?: string | undefined,
+    availability?: RawSchemas.DeclarationWithoutDocsSchema["availability"]
+): RawSchemas.ObjectPropertySchema {
```

Comment:
nit: we basically prefer objects as arguments in typescript because you are forced to name each parameter (e.g. kwargs) so its more readable